crates/cli/src/main.rs

```diff
@@ -107,12 +107,12 @@ async fn main() -> Result<()> {
                     println!("Getting Pool...");
 
                     // Get pool from CLI/defaults.
-                    let pool: Pool = get_pool(token0, token1, bp, provider).await.unwrap();
+                    let pool = get_pool(token0, token1, bp, provider).await.unwrap();
 
                     vec![pool]
                 }
             };
-            for pool in pools {
+            for mut pool in pools {
```

Comment:
why does this need to be mutable?

Reply:
Because monitor pools changes the pool by updating tick, liquidity and price.